.gitignore

```diff
@@ -1,7 +1,7 @@
 *.env.*
 .htaccess
 .idea/
-sites/*/sqlconf.php
-sites/*/profile_pictures/*
+sites/**
+!sites/default
```

Comment:
This is ok but we need not `default's sqlconfig`

Reply:
should i add !sites/*/sqlconf.php @muarachmann